elasticsearch/run.sh

```
@@ -85,7 +85,7 @@ wait_for_port_open() {
 do
     echo -n "."
     sleep $RETRY_INTERVAL
-    (( retry -= 1 ))
+    (( retry -= 1 )) || :
```

lets you ignore any errors basically